src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
+$card-size: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $card-size);
```

Comment:
Maybe prefer don`t use repeate one column. Anyway i`m don`t think that is mistake